src/style.css

```diff
@@ -0,0 +1,86 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  color: #fff;
+
+  --block-height: 300px;
+}
+
+.block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+  flex-grow: 1;
+  flex-basis: var(--block-height);
+  height: var(--block-height);
+}
+
+.block--1 {
+  background-color: rgb(100%,0%,0%);
+  order: 1;
+}
+
+.block--2 {
+  background-color: rgb(80%,0%,0%);
+  order: 2;
+}
+
+.block--3 {
+  background-color: rgb(60%,0%,0%);
+  order: 3;
+}
+
+.block--4 {
+  background-color: rgb(40%,0%,0%);
+  order: 4;
+}
+
+.block--5 {
+  background-color: rgb(20%,0%,0%);
+  order: 5;
+}
+
+.block--6 {
+  background-color: rgb(0,0,0);
+  order: 6;
```

Comment:
If you choose a mobile-first approach, your orders do nothing on mobile. Write minimum code for the mobile viewport (mobile-first) and then change the style on medias